src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -4889,6 +4894,10 @@ public sealed class EnablePSRemotingCommand : PSCmdlet
             if ($dotPos -ne -1) {{
                 $powershellVersionMajor = $powershellVersionMajor.Substring(0, $dotPos)
             }}
+            if ($PSVersionTable.PSVersion.ToString().Contains(""preview""))
```

Comment:
I don't see how appending "preview" gets the Issue `PowerShell.6.1.0-preview.3` endpoint name.  Where does the ending ".3" come from?

Reply:
I'll add a comment